plugins/zenoh-plugin-storage-manager/src/replica/aligner.rs

```diff
@@ -108,9 +109,10 @@ impl Aligner {
                 let Value {
                     payload, encoding, ..
                 } = value;
-                let sample = Sample::new(key, payload)
-                    .with_encoding(encoding)
-                    .with_timestamp(ts);
+                let sample = PutSampleBuilder::new(key, payload)
```

Comment:
From and API point of view, in my opinion would be better to have something like `SampleBuilder::put(key, payload) -> PutSampleBuilder`. But the user never explicitly uses a `PutSampleBuilder`. Opinions?

Reply:
agree, done